functions/README.md

```diff
@@ -0,0 +1,67 @@
+# SETUP CLOUD FUNCTIONS
+
+If you haven't initialized your firebase project, please start with setup
+stated in main directory README. This is only cloud functions guide.
+
+To deploy your functions, use
+
+```bash
+firebase deploy --only functions
+```
+
+## RUNNING LOCALLY
+
+Install the Firebase CLI
+
+```bash
+npm install -g firebase-tools
+```
+
+To run your functions locally, use
+
+```bash
+firebase emulators:start
+```
+
+## USING GOOGLE APIS CLOUD FUNCTIONS
+
+For Google APIs cloud functions such as gMail message sending and gSuite
+groups management, you need to have an active Google account and a domain with gSuite
+for business. Your account **has to be a superuser** on a gSuite platform you are accessing with cloud functions.
+Please fill the domain name and account email address in **config.json** file.
+
+To access your account functionalities programatically, you have to obtain a file
+with your credentials. The easiest way to do it is going here:
+https://developers.google.com/gmail/api/quickstart/js and clicking a button
+"Allow gMail API". The process will start and browser will automatically
```

Comment:
There are a lot of workflows to obtain this refresh token. You can click the button on the site I linked, it will create credentials.json and then in CLI (after running the app for the first time) you should be prompted to go to your account settings and allow the app to access GMail, GSuite etc. It shouldn't create a new GCP project, it didn't from my personal account's perspective.

You can also do it beforehand using the method you described here in the comment. For personal accounts it seems easier to do it my way, because CLI guides you step by step on what to click and what to copy-paste, but I'll include this workflow in the documentation too.

Reply:
And for my case, after creating token.json and invoking functions I was prompted something like "Your account has disabled gmail api, go to this link and click enable" and "Your account has disabled admin sdk api, go to this link and click enable" so it was easier to me to just follow the link than search for that settings without a link.

Reply:
Done.

Reply:
```suggestion
"Enable the Gmail API". The process will start and browser will automatically
```